Samples/BatchSupport/BatchSupport/Labels.txt

```diff
@@ -0,0 +1,1000 @@
+0,tench, Tinca tinca
```

Comment:
is this the same labels.txt that is in the other squeezenet sample? can you share the file instead of duplicating it?